### PR TITLE
Animals table

### PR DESCRIPTION
**Pull request by DavidEspinoG**
## What is new?
- Add scripts to create vet_clinic database and animals table
- Add scripts to populate animals table with sample data
- Add example queries 
- Update readme.md

### Queries 
- Find all animals whose name ends in "mon".
Captura de pantalla 2023-05-01 a la(s) 18 12 44
- List the name of all animals born between 2016 and 2019.
Captura de pantalla 2023-05-01 a la(s) 18 16 16
- List the name of all animals that are neutered and have less than 3 escape attempts.
Captura de pantalla 2023-05-01 a la(s) 18 18 11
- List the date of birth of all animals named either "Agumon" or "Pikachu".
Captura de pantalla 2023-05-01 a la(s) 18 19 54
- List name and escape attempts of animals that weigh more than 10.5kg
Captura de pantalla 2023-05-01 a la(s) 18 22 40
- Find all animals that are neutered.
Captura de pantalla 2023-05-01 a la(s) 19 02 45
-Find all animals not named Gabumon. 
Captura de pantalla 2023-05-01 a la(s) 18 25 30
- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3
Captura de pantalla 2023-05-01 a la(s) 18 27 17
kg)

### CODE REVIEW

**Review by Reem-lab — COMMENTED** [review]
# Hi @DavidEspinoG, 👋🏻👋🏻

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that you should delete the live demo section since you don't have the deployed link, it is not required to deploy it, so now just I am asking to delete the live demo section, once you deploy it you can add the section 🔥🤩

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by DavidEspinoG** [comment]
Thanks @Reem-lab, I think that change is a perfect example of an optional change on a code review. Cheers

File: README.md [hunk 1]
```diff
@@ -240,52 +140,34 @@ Example:
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/DavidEspinoG/vet_clinic_microverse/issues).
```
Comment (1GORDON) [inline README.md]: Great job @DavidEspinoG 🤝

- [ ] Kindly fix the broken issues link to make your readme file more professional.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @DavidEspinoG ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Descriptive Pull request ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @DavidEspinoG ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**